src/Spout/Writer/Common/Entity/Cell.php

```diff
@@ -55,12 +57,29 @@ class Cell
     protected $type = null;
 
     /**
+     * The cell style
+     * @var Style|null
+     */
+    protected $style = null;
+
+    /**
+     * @var StyleMerger
+     */
+    protected $styleMerger;
+
+    /**
      * Cell constructor.
      * @param $value mixed
+     * @param $style Style
      */
-    public function __construct($value)
+    public function __construct($value, Style $style = null)
     {
         $this->setValue($value);
+        if ($style) {
+            $this->setStyle($style);
+        }
+
+        $this->styleMerger = new StyleMerger();
```

Comment:
This should be injected through a factory. See comment below about CellManager.
You can instantiate a new Cell using the `EntityFactory` and the CellManager using the `ManagerFactory`